samples/bluetooth/mesh/light_dimmer/README.rst

```diff
@@ -0,0 +1,194 @@
+This mobile application is also used to configure key bindings, and publication and subscription settings of the Bluetooth mesh model instances in the sample.
+After provisioning and configuring the mesh models supported by the sample in the `nRF Mesh mobile app`_, Button 1 on the Light Dimmer device DK can be used to control
+the configured network nodes' LEDs.
+
+Provisioning
+============
+
+The provisioning is handled by the :ref:`bt_mesh_dk_prov`.
+It supports four types of out-of-band (OOB) authentication methods, and uses the Hardware Information driver to generate a deterministic UUID to uniquely represent the device.
+
+Models
+======
+
+The following table shows the composition data for the Light Dimmer sample:
+
+.. table::
+   :align: center
+
+   +-------------------------------+
+   | Element 1                     |
+   +===============================+
+   | Config Server                 |
+   +-------------------------------+
+   | Health Server                 |
+   +-------------------------------+
+   | Gen. Level Client             |
+   +-------------------------------+
+   | Gen. OnOff Client             |
+   +-------------------------------+
+
+The models are used for the following purposes:
+
+* The first element contains a Config Server and a Health Server.
+  The Config Server allows configurator devices to configure the node remotely.
+  The Health Server provides ``attention`` callbacks that are used during provisioning to call your attention to the device.
+  These callbacks trigger blinking of the LEDs.
+* The third model in the first element is the Generic Level Client.
+  The Generic Level Client controls the Generic Level Server in the target devices, deciding on parameters such as fade time and lighting level.
+* The last model is the Generic OnOff Client which controls the Generic OnOff Server in the target devices, setting the LED state to ON or OFF.
+
+
+The model handling is implemented in :file:`src/model_handler.c`, which uses the :ref:`dk_buttons_and_leds_readme` library to control the buttons on the development kit.
+
+User interface
+**************
+
+Buttons:
+
+* Can be used to input the out-of-band (OOB) authentication value during provisioning. All buttons have the same functionality during this procedure.
+* On press and release, Button 1 will publish a Gen. OnOff Set message using the configured publication parameters of its model instance and toggles the LED on/off on a :ref:`mesh light fixture <bluetooth_mesh_light_lc>` device.
+* On press and hold, Button 1 will publishe a Gen. Move Set message using the configured publication parameters of its model instance and will continuously dim the LED lightness state on a :ref:`mesh light fixture <bluetooth_mesh_light_lc>` device.
+  On release, the button client publish another Gen. Move Set message with the ``delta`` parameter set to 0, making the continuous level change from the previous message stop.
```

Comment:
I would suggest to change the layout on this to something like this:
Buttons:
   Can be used to input the out-of-band (OOB) authentication value during provisioning. All buttons have the same functionality during this procedure.

Button 1:
   On press and release will publish a Gen. OnOff Set message using the configured publication parameters of its model instance and toggles the LED on/off on a :ref:`mesh light fixture <bluetooth_mesh_light_lc>` device.

   On press and hold will publishe a Gen. Move Set message using the configured publication parameters of its model instance and will continuously dim the LED lightness state on a :ref:`mesh light fixture <bluetooth_mesh_light_lc>` device. 
   On release client publish another Gen. Move Set message with the ``delta`` parameter set to 0, making the continuous level change from the previous message stop.